### PR TITLE
DOC: Shared doc string in render methods

### PR DESCRIPTION
**Pull request by attack68**

### CODE REVIEW

File: pandas/io/formats/style.py [hunk 1]
```diff
@@ -97,6 +97,15 @@ def _mpl(func: Callable):
 color = """color : str, default 'yellow'
            Background color to use for highlighting."""
 
+buf = """buf : str, path object, file-like object, optional
+         String, path object (implementing ``os.PathLike[str]``), or file-like
+         object implementing a string ``write()`` function. If `None`, the result is
```
Comment (mroeschke) [inline pandas/io/formats/style.py:102]: Nit: Think None could use double ticks as well?

**Review by mroeschke — COMMENTED** [review]
Small comment otherwise LGTM

File: pandas/io/formats/style.py (hunk 1 above)
Comment (attack68) [inline pandas/io/formats/style.py:102]:
```suggestion
         object implementing a string ``write()`` function. If ``None``, the result is
```

**Review by mroeschke — APPROVED** [review]

**Comment by jreback** [comment]
thanks @attack68

**Pull request merged**